### ISSUE TITLE
Go to definition to package procedure in folder

### ISSUE THREAD

**Issue by mickeypearce** [zabel-xyz/plsql-language#31]
I have two packages in object type folders. Let's say:
```
./src/PACKAGES/PCK_1.sql
./src/PACKAGES/PCK_2.sql
```
with package definitions and
```
./src/PACKAGE_BODIES/PCK_1.sql
./src/PACKAGE_BODIES/PCK_2.sql
```
with package bodies.

I'd like to jump from procedure call in ./src/PACKAGE_BODIES/PCK_1.sql ex. 

> PCK_2.my_procedure()

 to my_procedure in ./src/PACKAGE_BODIES/PCK_2.sql.

I also tried with setting parameter 

`"plsql-language.searchFolder": "./src/PACKAGE_BODIES"`

but it doesn't seem to work.

It would be great if I could make it work. 
Thanks a lot.

**Comment by zabel-xyz** [comment]
It works fin for me (tested on system Windows 10)

![plsql_navigation](https://user-images.githubusercontent.com/16486407/32988898-dfde3f34-cd0d-11e7-8ae3-007e36190e3a.gif)

I upload files to test [here](https://github.com/zabel-xyz/plsql-language/tree/master/test/sql/src)

You could specify setting **searchFolder** (better performance),
but the folder must be **commun** to spec and bodies and also **absolute**

I'll improve this, relative path would be usefull !

**Comment by zabel-xyz** [comment]
You can use variable ${workspaceFolder}, in your case:
`"plsql-language.searchFolder": "${workspaceFolder}/src"`

**Comment by mickeypearce** [comment]
Hi,

The problem is with my package name declaration. I have schema username and package name in double quotes (this is Oracle's default export DDL setting for an object):

`CREATE OR REPLACE PACKAGE BODY "USER"."PCK_1"`

It would be great if it would at least work with double quotes:
`CREATE OR REPLACE PACKAGE BODY "PCK_1"`

Although considering schema (user) maybe isn't a bad idea.

**Comment by zabel-xyz** [comment]
I consider already schema, but not with double quotes so I'll fix this...

**Comment by zabel-xyz** [comment]
Fix in version 1.1.2

**Comment by mickeypearce** [comment]
Thanks, it works like a charm!
